resources/assets/js/components/Breadcrumbs/BreadcrumbsItem.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+
+interface BreadcrumbItemProps {
+  href: string;
+  title: string;
+}
+
+const BreadcrumbItem: React.FunctionComponent<BreadcrumbItemProps> = ({
+  href,
+  title,
+}): React.ReactElement => {
+  return (
+    <li>
+      <a href={href} title={title}>
```

Comment:
Looks like, for accessibility, the current page needs to have a `aria-current="page"` property.